.github/workflows/main.yml

```diff
@@ -248,8 +248,43 @@ jobs:
       - name: Test
         run: pytest
 
+  test-macos-arm64:
+    runs-on: flyci-macos-large-latest-m1
```

Comment:
If things run correctly, I will add a condition here like
```
if: (github.event_name == 'push') || (github.event_name == 'workflow_dispatch')
```

Reply:
So it will run only if we publish

Reply:
right... i forgot